src/solvers/solveCobraLPCPLEX.m

```diff
@@ -117,7 +117,10 @@
 if ~exist('minNorm','var')
     minNorm=0;
 end
-
+if ~printLevel && isunix

```

Comment:
`printLevel` is an integer, not a boolean.

`if printLevel == 0` (no need for testing the `OS`, as `cpxControl` is a `MATLAB` structure)